docs/source/markdown/options/cpu-period.md

```diff
@@ -5,8 +5,8 @@
 #### **--cpu-period**=*limit*
 
 Set the CPU period for the Completely Fair Scheduler (CFS), which is a
-duration in microseconds. Once the container's CPU quota is used up, it will
-not be scheduled to run until the current period ends. Defaults to 100000
+duration in microseconds. Once the container's CPU quota is used up, it is not
+scheduled to run until the current period ends. Defaults to 100000
```

Comment:
This too, I believe, even if more nuanced, should be a future tense (once ... it will not).